sdk/src/client/node_api/core/routes.rs

```diff
@@ -86,9 +87,10 @@ impl ClientInner {
     }
 
     /// Checks if the account is ready to issue a block.
-    /// GET /api/core/v3/accounts/{accountId}/congestion
+    /// GET /api/core/v3/accounts/{bech32Address}/congestion
     pub async fn get_account_congestion(&self, account_id: &AccountId) -> Result<CongestionResponse> {
-        let path = &format!("api/core/v3/accounts/{account_id}/congestion");
+        let bech32_address = account_id.to_bech32(self.get_bech32_hrp().await?);
+        let path = &format!("api/core/v3/accounts/{bech32_address}/congestion");
```

Comment:
hmm ... the method signature is not correct yet on purpose, right?

Reply:
What do you mean ?

Reply:
Maybe it should accept a bech32 address?

Reply:
I don't know tbh, I think I prefer the IDs 🤷🏻‍♂️